pom.xml

```diff
@@ -152,6 +157,7 @@
         <configuration>
           <debug>5008</debug>
           <suspend>false</suspend>
+          <systemProperties>clowder.</systemProperties>
```

Comment:
Is that required to enable the config source?

Reply:
For me it's also not that clear.

Reply:
No. That is a stupid leftover from a time when things did not fully add up yet